pennylane/beta/devices/numba_ops.py

```diff
@@ -0,0 +1,414 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Utility functions and numerical implementations of quantum operations for Numba-based devices.
+"""
+
+try:
+    from numba import jit
+except ImportError as e:
```

Comment:
```suggestion
except ImportError as e: #pragma: no-cover
```

If the line is something that can't be tested by the CI and code cov is complaining, you can add `#pragma: no-cover` to silence the code coverage warnings.